### DISCUSSION TITLE
Materialized Views in Table Editor

### DISCUSSION THREAD

**Discussion by michaeldever** [supabase/supabase#6806] [Questions]
I have just imported a data dump. Everything worked fine once I got around some issues with extensions.

The data import contained materizalized views, which I cannot see in the Table Editor. The data is in Supabase though, as I can select the dat from those views in the CLI pane, and can call functions to refresh them.

Would appreciate any advice on how to make those views visible, as having a view of that data is important.

**Comment by GaryAustin1** [comment]
Any Postgres database tool like pgadmin or datagrip works with the Postgres user port.
